access-control-app/src/cmr/access_control/api/routes.clj

```diff
@@ -368,7 +368,11 @@
 
         (GET "/"
              {ctx :request-context params :params}
-             (get-permissions ctx params)))
+             (get-permissions ctx params))
+
+        (POST "/"
```

Comment:
is the idea that they will be submitting a post request body? or are they wanting to hit a POST endpoint while using query params?

Reply:
Yes, you're right. I misunderstood the requirement. I've made the change to use a POST request body for providing the parameters.